### PR TITLE
Migration case attachments

### PR DESCRIPTION
**Pull request by rjweeks**
### HOCS-5584

Add case attachments (documents) to Document Service.

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/dto/CaseAttachment.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+@Getter
+public class CaseAttachment {
+
+    @JsonProperty("displayName")
+    private String displayName;
+
+    @JsonProperty("documentType")
+    private String type;
+
+    @JsonProperty("s3UntrustedUrl")
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/dto/CaseAttachment.java:21]: As commented on the other one - I think this probably should be `documentUuid` or similar

Comment (SWhitlam) [reply src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/dto/CaseAttachment.java:21]: > As commented on the other one - I think this probably should be `documentUuid` or similar

I don't think this is the document UUID though? It's the URL of the file in the untrusted bucket which happens to be a UUID?

Comment (LiamMacP) [reply src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/dto/CaseAttachment.java:21]: `documentPath` then? Or even `fileLink` as you suggested? I don't see the value of having `s3Untrusted` nor `Url` as its not strictly accurate

Comment (rjweeks) [reply src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/dto/CaseAttachment.java]: Updated.

**Pull request closed without merging**